src/PAModel/Serializers/MsAppSerializer.cs

```diff
@@ -527,6 +549,25 @@ private static IEnumerable<FileEntry> GetMsAppFiles(this CanvasDocument app, Err
             var props = app._properties.JsonClone();
             if (app._connections != null)
             {
+                var connectionIDReferences = app._entropy.LocalConnectionIDReferences;
+                if (connectionIDReferences != null)
```

Comment:
I'm guessing this handles the case for backwards compat? Meaning if an older unpacked App does not have this entropy value, this will come back as `null` and be skipped?

Reply:
> I'm guessing this handles the case for backwards compat? Meaning if an older unpacked App does not have this entropy value, this will come back as `null` and be skipped?

I added this as an extra validation.

Reply:
We talked through this - back compat will keep the `connectionInstanceId` as what it was before this change, which is even better.